### ISSUE TITLE
Nitpick about ~zod version number and release notes

### ISSUE THREAD

**Issue by msutherl** [urbit/urbit#1394]
`zod.YYYY.M.D` doesn't make too much sense syntactically. How about `~zod-YYYY.M.D`?

<img width="995" alt="Screenshot 2019-07-26 11 0 26" src="https://user-images.githubusercontent.com/31774/61971735-a2088a00-af94-11e9-8e8f-6a1737ca261f.png">

Also, for someone browsing the release page, they might be confused as to why there is a 'zod release' there. "Do I need to install that?" Perhaps a short description in the release notes about what it's for, and what "OTA" means.

**Comment by Fang-** [comment]
Tangentially related: we can and should have tags for every single time we've pushed changes to ~zod, ever.

If I want to replay the rescue efforts from this week for research purposes, I currently have to bother the experts involved to find out what happened, instead of just looking at the tags and finding `zod.2019.7.25-1` through `zod.2019.7.25-4` or similar.

I realize this is low priority when events are ongoing, but feel like this should be somewhere in the "doing things OTA" timeline.

**Comment by ixv** [comment]
Ah, I just made another release yesterday to reflect the changes me and Philip made, without reading this issue. I agree this is sensible. What do you think of making this our standard copy:


> This release reflects changes pushed to the live network for an over-the-air update. Users on the live network will get this update automatically, and need not take any action
> 
> [specific release notes if applicable]
> [list of PRs in this release]

Also agree that the name is bad. It probably shouldn't say zod at all, but perhaps something like `ota-YYYY.M.D`

**Comment by ixv** [comment]
pinging @jtobin and @joemfb

**Comment by Fang-** [comment]
> It probably shouldn't say zod at all, but perhaps something like ota-YYYY.M.D

But it's going OTA through ~zod, right? If you're not (ultimately) getting your updates from ~zod, then you won't match these dates/contents. We don't control the entire network, and our naming/messaging should reflect that. (Agreed that `zod-ota-timestamp` might be clearer though.)

**Comment by jtobin** [comment]
> Tangentially related: we can and should have tags for every single time we've pushed changes to ~zod, ever.

Definitely.  As a corollary: we should only ever push changes OTA that correspond to some tag (via some mechanism or other).

> Also agree that the name is bad. It probably shouldn't say zod at all, but perhaps something like ota-YYYY.M.D

Agreed -- I suggested the `zod-YYYY.M.D` format as a placeholder for the time being, but it should be changed.

In general releases for OTA updates should consist of commits, cherry-picked and applied on top of the previous tag, then tagged appropriately in some branch.  During my brief experience with this sort of thing last era, I found it conceptually useful to have some indicator in the tag of what era you're supposed to be on.

I would prefer something that ignored the date (which is of course available in the tag anyway) and simply counted up simpler version numbers in some way.  My original idea was to use e.g. `0.8.1`, `0.8.2`, etc., where the `0.8` prefix refers to the current era (and is gotten from the latest breaching release); @joemfb expressed concerns about the effect on UX by just bumping the "patch version" like that, so this all deserves some extra thought in any case.  I'll get to thinking..
